src/Slider/TimeSlider/TimeSlider.tsx

```diff
@@ -52,7 +52,7 @@ export interface BaseProps {
   marks?: SliderMarks;
 }
 
-export type TimeSliderProps = BaseProps & Partial<DefaultProps> & SliderProps;
```

Comment:
Having the `TimeSliderProps` extend the `SliderProps` seems very legit to me, why did you remove them?

Reply:
We have removed these, because otherwise an error occurs with `timeLayerAware(TimeSlider, this._wmsTimeLayers);`: `Argument of type 'typeof TimeSlider' is not assignable to parameter of type 'ComponentType<TimeSliderProps>'`. This could only be bypassed for compilation with the specification `\\ @ts-ignore`

Reply:
The doubled props defined by the antd slider are omitted.